src/schemas/components/SchemaFormPropertyArrayItem.vue

```diff
@@ -6,7 +6,7 @@
       <component :is="input.component" v-bind="input.props" />
     </keep-alive>
 
-    <SchemaFormPropertyMenu v-model:kind="kind">
+    <SchemaFormPropertyMenu v-model:kind="kind" :property="property">
```

Comment:
i've been wondering when/if we can use the bind shorthand now... slash what thing we need to flip to be able to

Reply:
oh yeah! That will be nice. I wonder if volar has caught up 🤔